tests/ha_master/run.sh

```diff
@@ -248,6 +248,7 @@ function test_list_member() {
 }
 
 function run() {
+	run_sql_both_source "SET @@GLOBAL.SQL_MODE=''"
```

Comment:
why add this line

Reply:
[`ERROR 1064 (42000) at line 8: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '"$.id")' at line 1`](https://ci.pingcap.net/blue/organizations/jenkins/dm_ghpr_new_test/detail/dm_ghpr_new_test/737/pipeline/149)

this failed caused by sqlmode not right ?

Reply:
yes, but in case "ha_master" we didn't change sql_mode, and the cases are run independently in my mind. we should investigate why the sql_mode is changed to a bad value (maybe 'ANSI_QUOTES')

Reply:
this is beacuse we reuse test pod in ci, and other test such as `all_mode` will set sqlmode to `ANSI_QUOTES,NO_AUTO_VALUE_ON_ZERO`

Reply:
we should reset the sql_mode after finishing `all_mode`, or reset before any case starting. Only resetting here is not acceptable.

And I'm not sure if we have reuse `golang` pod, or pods can be accessedrun concurrently 😵 cc @GMHDBJD @glorv

Reply:
ok  i will fix this